kratos/solving_strategies/builder_and_solvers/explicit_builder.h

```diff
@@ -0,0 +1,846 @@
+    {
+        return mCalculateReactionsFlag;
+    }
+
+    /**
+     * @brief This method sets the flag mCalculateReactionsFlag
+     * @param CalculateReactionsFlag The flag that tells if the reactions are computed
+     */
+    void SetCalculateReactionsFlag(bool CalculateReactionsFlag)
+    {
+        mCalculateReactionsFlag = CalculateReactionsFlag;
+    }
+
+    /**
+     * @brief This method returns the flag mDofSetIsInitialized
+     * @return The flag that tells if the dof set is initialized
+     */
+    bool GetDofSetIsInitializedFlag()
+    {
+        return mDofSetIsInitialized;
+    }
+
+    /**
+     * @brief This method sets the flag mDofSetIsInitialized
+     * @param DofSetIsInitialized The flag that tells if the dof set is initialized
+     */
+    void SetDofSetIsInitializedFlag(bool DofSetIsInitialized)
+    {
+        mDofSetIsInitialized = DofSetIsInitialized;
+    }
+
+    /**
+     * @brief This method returns the flag mReshapeMatrixFlag
+     * @return The flag that tells if we need to reset the DOF set
+     */
+    bool GetResetDofSetFlag()
+    {
+        return mResetDofSetFlag;
+    }
+
+    /**
+     * @brief This method sets the flag mResetDofSetFlag
+     * @param mResetDofSetFlag The flag that tells if we need to reset the DOF set
+     */
+    void SetResetDofSetFlag(bool ResetDofSetFlag)
+    {
+        mResetDofSetFlag = ResetDofSetFlag;
+    }
+
+    /**
+     * @brief This method returns the flag GetResetLumpedMassVectorFlag
+     * @return The flag that tells if we need to reset the lumped mass vector
+     */
+    bool GetResetLumpedMassVectorFlag()
+    {
+        return mResetLumpedMassVectorFlag;
+    }
+
+    /**
+     * @brief This method sets the flag mResetLumpedMassVectorFlag
+     * @param ResetLumpedMassVectorFlag The flag that tells if we need to reset the lumped mass vector
+     */
+    void SetResetLumpedMassVectorFlag(bool ResetLumpedMassVectorFlag)
+    {
+        mResetLumpedMassVectorFlag = ResetLumpedMassVectorFlag;
+    }
+
+    /**
+     * @brief This method returns the value mEquationSystemSize
+     * @return Size of the system of equations
+     */
+    unsigned int GetEquationSystemSize()
+    {
+        return mEquationSystemSize;
+    }
+
+    /**
+     * @brief It allows to get the list of Dofs from the element
+     */
+    virtual DofsArrayType& GetDofSet()
+    {
+        return mDofSet;
+    }
+
+    /**
+     * @brief Get the lumped mass matrix vector pointer
+     * It allows to get the lumped mass matrix vector pointer
+     * @return TSystemVectorPointerType& The lumped mass matrix vector pointer
+     */
+    virtual TSystemVectorPointerType& pGetLumpedMassMatrixVector()
+    {
+        return mpLumpedMassVector;
+    }
+
+    /**
+     * @brief Get the lumped mass matrix vector
+     * It allows to get the lumped mass matrix vector
+     * @return TSystemVectorType& The lumped mass matrix vector
+     */
+    virtual TSystemVectorType& GetLumpedMassMatrixVector()
```

Comment:
same

plus at least in debug you should check if it is a `nullptr` otherwise it segfaults!